demo/elastic/wazuh_cf_kibana.sh

```diff
@@ -444,12 +427,11 @@ main(){
   enable_kibana
   optimize_kibana
   start_kibana
-  sleep 60
+  sleep 200
```

Comment:
Please explain why this is necessary.

Reply:
During kibana optimize, the `wazuh.yml` file of the Wazuh APP is copied and after finishing the optimize it is mistakenly replaced with a default one. 

This sleep allows the optimize to finish and then the `wazuh.yml` file is copied in time, avoiding being replaced by the default one.